### DISCUSSION TITLE
pull request

### DISCUSSION THREAD

**Discussion by CodeCraftsman20** [community/community#142331] [Pull Requests]
### Select Topic Area

Question

### Body

How do I resolve merge conflicts in a pull request on GitHub

**Comment by CodewithShahriar** [comment]
When a pull request has merge conflicts, GitHub will highlight them. I clone the branch locally, resolve conflicts in the affected files, commit the changes, and push the updated branch. After that, the pull request can be merged without issues.

**Reply by CodeCraftsman20** [reply]
Thanks

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/142329#discussioncomment-11022784) for reference to why this discussion was closed and locked. Thanks!
